docs/apache-airflow/administration-and-deployment/logging-monitoring/metrics.rst

```diff
@@ -182,6 +182,7 @@ Name                                                                   Descripti
 ``task_restored_to_dag.<dag_id>``                                      Number of tasks restored for a given dag (i.e. task instance which was
                                                                        previously in REMOVED state in the DB is added to DAG file)
 ``task_instance_created_<operator_name>``                              Number of tasks instances created for a given Operator
+``scheduler_heartbeat``                                                Triggerer heartbeats
```

Comment:
copy/pasta
```suggestion
``triggerer_heartbeat``                                                Triggerer heartbeats
```

Reply:
LOL! Two lines change and I have not even noticed that. Thanks for the catch!